bundle/Controller/ContentEditController.php

```diff
@@ -88,10 +92,9 @@ public function createWithoutDraftAction($contentTypeIdentifier, $language, $par
             }
         }
 
-        return $this->render('EzSystemsRepositoryFormsBundle:Content:content_edit.html.twig', [
+        return new ContentEditView(null, [
             'form' => $form->createView(),
             'languageCode' => $language,
-            'pagelayout' => $this->pagelayout,
```

Comment:
Is this setting passed to the view somewhere else? I'm not a Twig expert, just asking :)

Reply:
Yes. `pagelayout` is now passed via `ViewTemplatesListener`. This allows us to attach different pagelayout for admin UI.

Reply:
Ah, ok, I see now :)